### PR TITLE
Added instructions to build KLEE with TCMalloc support

### PR DESCRIPTION
**Pull request by andreamattavelli**
To fix issue #41 
@MartinNowack may I ask you to check if the dependencies are correct?

### CODE REVIEW

File: build-llvm34.md [hunk 1]
```diff
@@ -23,6 +23,12 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
    $ export CPLUS_INCLUDE_PATH=/usr/include/x86_64-linux-gnu
    ```
 
+   **(Optional) Build KLEE with TCMalloc support:** By default, KLEE uses malloc_info() to get memory usage, which limits the maximum memory to 2000MB. To avoid this limitation, it is possible to also configure KLEE with TCMalloc. It it thus necessary to install TCMalloc:
```
Comment (MartinNowack) [inline build-llvm34.md]: ... KLEE uses `malloc_info()` to observe and to restrict its memory usage. Due to limitations of `malloc_info()` [maybe a link here], the maximum limit set is 2 GB. To support bigger limits, KLEE can use TCMalloc as an alternative allocator.
It *is* ..."

File: build-llvm34.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ If you want to build KLEE with LLVM 2.9, [click here]({{site.baseurl}}/build-llv
...
+
+   ```bash
+   $ sudo apt-get install libtcmalloc-minimal4 libgoogle-perftools-dev libcap-dev
```
Comment (MartinNowack) [inline build-llvm34.md]: libcap-dev is not a requirement for tcmalloc.
Everything else looks fine.

**Comment by MartinNowack** [comment]
@andreamattavelli Thanks a lot! Great to see that in the manual.

**Comment by andreamattavelli** [comment]
@MartinNowack updated with your comments. Please check that this time everything is fine.

File: build-llvm29.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ If you want to build KLEE with LLVM 3.4 (recommended), [click here]({{site.baseu
    $ export CPLUS_INCLUDE_PATH=/usr/include/x86_64-linux-gnu
    ```
 
+   **(Optional) Build KLEE with TCMalloc support:** By default, KLEE uses malloc_info() to observe and to restrict its memory usage. Due to limitations of malloc_info(), the maximum limit set is 2 GB. To support bigger limits, KLEE can use TCMalloc as an alternative allocator. It it thus necessary to install TCMalloc:
```
Comment (MartinNowack) [inline build-llvm29.md]: "It it thus" -> "It is thus"

**Comment by MartinNowack** [comment]
Beside the typo, I'm absolutely happy with it.

**Comment by andreamattavelli** [comment]
> Beside the typo

Thanks for pointing out the typo!

**Comment by ccadar** [comment]
One suboptimal thing here is that the same text is duplicated for each LLVM version we support.  I guess it's fine, although another option would be to create a separate page in such cases.

**Comment by andreamattavelli** [comment]
> One suboptimal thing here is that the same text is duplicated for each LLVM version we support. I guess it's fine, although another option would be to create a separate page in such cases.

I thought about creating a separate page like in the case of STP, but I then realized that it was too much for such a "small" detail (6 lines counting also the empty ones). I thus went for the replication.

**Pull request merged**